docs/protocol/index.md

```diff
@@ -0,0 +1,63 @@
+# Protocol
+
+## Overview
+
+---
+
+Ceramic is a decentralized event streaming protocol that enables developers to build decentralized databases, distributed compute pipelines, and authenticated data feeds, etc. Ceramic nodes can subscribe to subsets of streams forgoing the need of a global network state. This makes Ceramic an eventually consistent system (as opposed to strongly consistent like L1 blockchains), enabling web scale applications to be built reliably.
```

Comment:
Could clarify a bit here about how forgoing global network state affect data availability? Who is mainly responsible for keeping the data available? If third party developer shut down their own ceramic node, who else is pining the data and why?

Reply:
> If third party developer shut down their own ceramic node, who else is pining the data and why?

Right now there is no global incentive layer to keep data available. If there is data in the network that you really care about you need to run a node that subscribes to this data so that it's kept available.

In the future we intend to work on an incentive layer to solve this issue.